### PR TITLE
fix: classroom card appearance

### PR DESCRIPTION
**Pull request by jfdoming**
## Notion ticket link

[Fix classroom card](https://www.notion.so/uwblueprintexecs/Fix-classroom-card-fc6b90e96dc94d8bba38dbb0a17038da)

## Implementation description
See ticket and comment below. Also please look at the commits and messages on the commits!

## Steps to test
1. View card
2. Distribute an assessment and select a card

## What should reviewers focus on?
* 

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by jfdoming** [comment]
Question: currently, in the assessment list, "active" assessments are green and "upcoming" assessments are yellow. However, we're now marking `ClassroomCard` "active" assessments as yellow. This is inconsistent; however, using green instead of yellow on `ClassroomCard`s looks pretty bad when hovered (low contrast):

image

Wondering if you have any alternative ideas for how to fix this problem?

**Comment by carissa-tang** [comment]
i feel like having consistent color (green for active) would make more sense. could we try adding a dark green border or shadow to the tag (on hover, or always)?

File: backend/middlewares/auth.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const isAuthorizedByRole = (roles: Set<Role>) => {
 /* Determine if request for a user-specific resource is authorized based on accessToken
  * validity and if the userId that the token was issued to matches the requested userId
  * Note: userIdField is the name of the request parameter containing the requested userId */
-export const isAuthorizedByUserId = (userIdField: string) => {
```
Comment (carissa-tang) [inline backend/middlewares/auth.ts:65]: update the doc with the new name?

**Comment by jfdoming** [comment]
pt: keeping green low-contrast text

**Review by carissa-tang — APPROVED** [review]
nice

**Pull request merged**